tests/random_test.py

```diff
@@ -200,9 +200,11 @@ def testRngRandint(self, dtype):
       self.assertTrue(np.all(samples < hi))
 
   @parameterized.named_parameters(jtu.cases_from_list(
-      {"testcase_name": "_{}".format(dtype), "dtype": np.dtype(dtype).name}
-      for dtype in [np.float32, np.float64]))
+      {"testcase_name": "_{}".format(dtype), "dtype": np.dtype(dtype)}
+      for dtype in [np.float16, np.float32, np.float64]))
```

Comment:
You might consider also testing `bfloat16`, which is supported on all of our platforms (if slow on some)?

Reply:
I initially had bfloat16 here, but removed it because it caused test failures due to low precision not passing the statistical tests below.